ui/pages/settings/transaction-shield-tab/claims-list/claims-list.tsx

```diff
@@ -67,9 +72,9 @@ const ClaimsList = () => {
   const { pendingClaims, historyClaims, isLoading } = useClaims();
 
   const claimItem = useCallback(
-    (claim: ShieldClaim) => {
+    (claim: Claim) => {
       // add leading zero to claim number if it is less than 1000
-      const claimNumber = claim.claimNumber.toString().padStart(3, '0');
+      const claimNumber = claim.id.toString().padStart(3, '0');
```

Comment:
did you change the id to be a counter on claims ctrl? because the id is too long to show on UI eg. `05fa245a-1a4d-427a-b4bb-c8d409c05c38`

we want to show it like `Claims #1`... on UI.